prog-o-meter.py

```diff
@@ -215,90 +279,103 @@ def __init__(self, _user_type):
-            self.canvas.create_text(self.CANVAS_WIDTH/2, 20, text = "Good to see you again! Please enter your name")
+            self.canvas.create_text(self.CANVAS_WIDTH/2, 20, text="Good to see you again! Please enter your name")
         elif self.user_type == 2:        # Display appropriate greeting for new users
-            self.canvas.create_text(self.CANVAS_WIDTH/2, 20, text = "Lets get you started! Please enter your name")
+            self.canvas.create_text(self.CANVAS_WIDTH/2, 20, text="Lets get you started! Please enter your name")
+            
     def input_button(self):
         """Display the inout button on the canvas.
-        
-        Displays a submit button, for user to click when they have typed in their name.
-        When button is clicked, it stores the input name as username, and then closes the Tkinter window.
-        
+
+        Displays a submit button, for user to click when they have typed
+        in their name.
+        When button is clicked, it stores the input name as username,
+        and then closes the Tkinter window.
+
         Attributes:
-            submit_button: Button with the text "Submit", which calls the function save_and_close
+            submit_button: Button with the text "Submit", which calls
+            the function save_and_close
         """
-        self.submit_button = Tk.Button(self.root, text = "Submit", command = self.save_and_close)
+        self.submit_button = Tk.Button(self.root, text="Submit", command=self.save_and_close)
         self.submit_button.pack()
+        
     def save_and_close(self):
         """Save input text as username, then close Tkinter window. """
         self.username = self.text_entry.get()
         self.root.destroy()
+        
     def get_name(self):
         """Return the username. """
         return self.username
-            
+
 def update_days_file(_filename, _days):
-    """Update the file [username].txt, with the current number of days completed.
-    
+    """Update the file [username].txt, with the current number of
+    days completed.
+
     Args:
-        _filename: Name of the file to be updated. Should have format [username].txt (username in all lowercase).
+        _filename: Name of the file to be updated. Should have format
+        [username].txt (username in all lowercase).
         _days: the current number of days completed
     """
     days_text = open(_filename, "w")
     days_text.write(str(_days))
     days_text.close()
-  
+
 def read_days_file(_filename):
-    """Read the file [username].txt, to retrieve the number of days completed, from last use.
-    
+    """Read the file [username].txt, to retrieve the number of days
+    completed, from last use.
+
     Args:
-        _filename: Name of the file to be read. Should have format [username].txt (username in all lowercase).
-    
+        _filename: Name of the file to be read. Should have format \

```

Comment:
What is the purpose of the \ here?

Reply:
Also, how to indicate that line 349 groups with line 348?

Reply:
I wasn't entirely sure what the \ was for, so I just left it.  It could have been a slip of mine on the keyboard to be honest.  I am heavily medicated at the moment.  As for the indication of the lines grouping, maybe indenting the second line a bit to say hey, this line follows?  I know often times the \ has been used to indicated the next line is part of this line, so maybe that is what I was thinking if I did that?

Reply:
Oh cool, I'll try and take a better look at PEP8 too, and see if I can find the standard for how to keep the lines grouped